qdrant-landing/content/documentation/guides/administration.md

```diff
@@ -7,7 +7,12 @@ aliases:
 
 # Administration
 
-Qdrant exposes administration tools which enable to modify at runtime the behavior of a qdrant instance without changing its configuration manually.
+Qdrant exposes administration tools which allow you to modify the behavior of a Qdrant instance at runtime:
+
+- [Locking](#locking)
+- [Recovery mode](#recovery-mode)
```

Comment:
Though I'm fine with listing these here, we also show them on the right in the table of contents.